src/plugins/auto/src/auto_schedule.cpp

```diff
@@ -446,6 +447,18 @@ void AutoSchedule::init(const ScheduleContext::Ptr& sContext) {
                 _loadContext[CPU].isEnabled = true;
                 _loadContext[CPU].deviceInfo = *CPUIter;
                 _loadContext[CPU].deviceInfo.config[CONFIG_KEY(PERFORMANCE_HINT)] = IE::PluginConfigParams::LATENCY;
+                // limit the compilation threads for the actual selected device
+                const int num_logic_cores = InferenceEngine::getNumberOfLogicalCPUCores();
```

Comment:
Use new API ov::get_number_of_logical_cpu_cores instead of InferenceEngine::getNumberOfLogicalCPUCores